<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewingWindow</class>
 <widget class="QMainWindow" name="ViewingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ViewingWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PIC 10B Grading Scheme</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>231</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>HW1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="hw2_slider_1">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hw1_score">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>HW2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="hw2_slider_2">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hw2_score_2">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>HW3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="hw2_slider_3">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hw2_score_3">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>HW4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="hw2_slider_4">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hw2_score_4">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>HW5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="hw2_slider_5">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hw2_score_5">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>HW6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="hw2_slider_6">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hw2_score_6">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>HW7</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="hw2_slider_7">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hw2_score_7">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>HW8</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="hw2_slider_8">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hw2_score_8">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>211</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Midterm 1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="horizontalSlider">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QSpinBox" name="spinBox_2">
               <property name="maximum">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Midterm 2</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QSlider" name="horizontalSlider_2">
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_3">
               <property name="maximum">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Final</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QSlider" name="horizontalSlider_3">
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_4">
               <property name="maximum">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>SchemeA</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Ascore">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>SchemeB</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Bscore">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>534</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>hw2_slider_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw1_score</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw1_score</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw2_slider_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw2_score_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw2_slider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw2_slider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw2_score_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw2_score_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw2_slider_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw2_slider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw2_score_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw2_score_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw2_slider_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw2_slider_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw2_score_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw2_score_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw2_slider_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw2_slider_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw2_score_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw2_score_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw2_slider_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw2_slider_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw2_score_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw2_score_7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw2_slider_7</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw2_slider_7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw2_score_7</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw2_score_8</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw2_slider_8</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>462</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>462</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
